ui/.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Can we double check if we can add this to the root pathway so everything is immediately usable after checking out the repo including running UI commands rather than having to cd into a nested directory?